code/parachain/frame/crowdloan-rewards/src/lib.rs

```diff
@@ -350,6 +382,63 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		/// Patch existing rewards by deleting them.
+		/// This will cause inconsistencies if a claim has already been made. Errors can be avoided
+		/// by passing the expected amount of rewards claimable.
+		///
+		/// Security:
+		/// * Should be called within a transactional context, as this may return an error but still
+		///   change
+		/// storage.
+		// pub(crate) fn do_delete(
+		// 	deletions: Vec<(RemoteAccountOf<T>, BalanceOf<T>)>,
+		// ) -> DispatchResult {
+		// 	let mut total_rewards: T::Balance = TotalRewards::<T>::get();
+		// 	let mut total_contributors: u32 = TotalContributors::<T>::get();
+
+		// 	// Get each account from deletions and remove it from the storage
+		// 	for (account, expected) in deletions {
+		// 		Rewards::<T>::mutate_exists(account, |rewards| {
+		// 			if let Some(rewards) = rewards {
+		// 				ensure!(rewards.total == expected, Error::<T>::UnexpectedRewardAmount);
+		// 				total_rewards -= rewards.total;
+		// 				total_contributors -= 1;
+		// 			}
+		// 			*rewards = None;
+		// 			Ok::<(), Error<T>>(())
+		// 		})?;
+		// 	}
+
+		// 	TotalRewards::<T>::set(total_rewards);
+		// 	TotalContributors::<T>::set(total_contributors);
+		// 	Ok(())
+		// }
```

Comment:
Maybe we should clarify why this is also commented out with the line used above in the other commented block: 
`// README: Disabled to reduce the amount of auditing work. When enabling, also enable the
		// tests.`
		
Not a biggy but why do we need to leave the commented code in?

Reply:
Agree, will just remove.